components/cards/cards.html

```diff
@@ -0,0 +1,275 @@
+							style="
+								width: 303px;
+								height: 200px;
+								border: 0;
+								transform: scale(1);
+								overflow: hidden;
+								margin-top: 2rem;
+								margin-bottom: 4rem;
+							"
+							sandbox="allow-scripts allow-same-origin"
+						>
+						</iframe>
+
+						<!-- card with shadow -->
+						<div class="intro-heading lg-text">Card with shadow</div>
+						<p class="intro-sub-heading sm-text">
+							To add shadow to any card add card-shadow class.
+						</p>
+						<div class="card card-shadow">
+							<img
+								src="../../assets/illustration-hero.svg"
+								alt="card-example"
+							/>
+							<h1>Card Shadow</h1>
+							<p class="sm-text">
+								Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+								Voluptates ut pariatur vel quod maxime necessitatibus!
+							</p>
+						</div>
+						<iframe
+							src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C1%29&t=seti&wt=none&l=auto&width=774&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=18px&ph=44px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2520card-shadow%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520src%253D%2522.%252Fyour-img%2522%2520alt%253D%2522card-example%2522%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%253Ch1%253ECard%2520shadow%2520example%253C%252Fh1%253E%250A%2509%253Cp%253E%2520Text%2520Here%2520%253C%252Fp%253E%250A%253C%252Fdiv%253E"
+							style="
+								height: 230px;
+								border: 0;
+								transform: scale(1);
+								overflow: hidden;
+								margin-top: 2rem;
+								margin-bottom: 4rem;
+							"
+							sandbox="allow-scripts allow-same-origin"
+						>
+						</iframe>
+
+						<!-- card with text overlay -->
+						<div class="intro-heading lg-text">Card with text overlay</div>
+						<p class="intro-sub-heading sm-text">
+							To overlay text over other items, you need to add text-overlay
+							class to the text which you want to overlay.
+						</p>
+						<div class="card card-overlay card-shadow">
+							<img
+								src="../../assets/illustration-hero.svg"
+								alt="card-example"
+							/>
+							<h1 class="text-overlay">Card Overlay</h1>
+							<p class="sm-text">
+								Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+								Voluptates ut pariatur vel quod maxime necessitatibus!
+							</p>
+						</div>
```

Comment:
The card title is not getting overlayed properly. You could fix this by positioning it a bit up.

![Screenshot 2022-02-02 at 18 19 15](https://user-images.githubusercontent.com/62737267/152157310-b74f0d2e-70e1-4740-8685-58a4c81eed62.png)

Reply:
It is looking fine in local.

![Screenshot (76)](https://user-images.githubusercontent.com/86295253/152161134-aef29e5e-a377-4f64-a500-1267c91f09e3.png)

Reply:
Ok, you might want to check it on the netlify deploy preview. Because it's still broken there.